packages/loader/container-loader/src/container.ts

```diff
@@ -1592,10 +1592,13 @@ export class Container
 			specifiedVersion,
 			supportGetSnapshotApi,
 		);
+		const baseSnapshotTree: ISnapshotTree | undefined = isInstanceOfISnapshot(baseSnapshot)
```

Comment:
In addition to having isInstanceOfISnapshot(), consider adding this helper:

getSnapshotTree(tree: ISnapshotTree | ISnapshot) => ISnapshotTree

Reply:
done